tests/e2e/features/transactions/transfer.feature

```diff
@@ -0,0 +1,26 @@
+@transfer @regression @transactions 
+Feature: Transfer
+
+  Background:
+    Given Connect Metamask extension with login action
+
+  @id1231
```

Comment:
Is this correct id? Allure: "Test case was deleted"

Reply:
Test number fixed to correct id1321